style.css

```diff
@@ -0,0 +1,85 @@
+@import url("https://fonts.googleapis.com/css?family=Poppins:200,300,400,500,600,700,800,900&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  background-color: #e5e5e5;
+}
+
+a {
+  text-decoration: none;
+  cursor: pointer;
+}
+
+li {
+  list-style-type: none;
+}
+
+/* intro style */
+.container-fluid {
+  position: relative;
+  top: 0;
+  bottom: 0;
+  width: 100%;
+  background-color: #fff;
+  border-bottom-left-radius: 60px;
+  box-shadow: 1px 0 1px #fff;
+}
+
+/* logo style */
+.display-flex1 {
+  display: flex;
+  justify-content: space-between;
+  padding: 1rem 3rem 0 0;
+}
+
+/* main content style */
+.main {
+  width: 100%;
+  padding: 1rem;
+  background-image: url("./images/header-shapes-mobile.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.display-flex2 {
+  display: flex;
+  flex-direction: column;
+}
+
+/* intro style */
+.intro {
+  margin-top: 3.5rem;
+}
+
+.h1 {
+  font-size: 3rem;
+  padding-bottom: 1rem;
+}
```

Comment:
- You don't have to use `.h1` class. You can just use `h1` tag and you can delete h1 class from html.

Reply:
Done